src/js/components/TextArea/TextArea.js

```diff
@@ -43,25 +43,26 @@ const TextArea = forwardRef(
           if (onKeyDown) {
             onKeyDown(event);
           }
-          if (ref?.current && !!ref.current.value === false) {
```

Comment:
I think the preferred way to do this would instead be `useForwardedRef`. It handles the situation where no ref was passed.

Also we'll need to discuss this feature further when more of the team is available.